projects/vdk-plugins/airflow-provider-vdk/vdk_provider/hooks/vdk.py

```diff
@@ -1,75 +1,110 @@
 # Copyright 2021 VMware, Inc.
 # SPDX-License-Identifier: Apache-2.0
+import logging
+
 from airflow.providers.http.hooks.http import HttpHook
+from taurus_datajob_api import DataJobExecutionRequest
 from tenacity import sleep
 from tenacity import stop_after_attempt
 from vdk.internal.control.auth.auth import Authentication
+from vdk.internal.control.rest_lib.factory import ApiClientFactory
+
+log = logging.getLogger(__name__)
 
 
 class VDKHook(HttpHook):
     def __init__(
         self,
-        conn_id,
-        job_name,
-        team_name,
-        timeout=1,  # TODO: Set reasonable defaults
-        retry_limit=1,
-        retry_delay=1,
+        conn_id: str,
+        job_name: str,
+        team_name: str,
+        timeout: int = 1,  # TODO: Set reasonable default
+        retry_limit: int = 3,
+        retry_delay: int = 10,
```

Comment:
There seemed to be unused variables here?

Reply:
dropped